tests/tests_integration/test_integration.py

```diff
@@ -16,6 +16,7 @@ def client():
 
     # Create the new app.
     test_app = app.create_app()
+    test_app.config['LOGIN_DISABLED'] = True
```

Comment:
Not a problem, but is there a reason to prefer doing it this way to setting the relevant environment variable in `.env.test`?